### PR TITLE
Fix Visual Inconsistencies in Item Collection

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary
This PR addresses https://github.com/cerner/terra-clinical/issues/169. 

Additionally, it updates then nightwatch tests to use expect syntax instead of assert.

### CODE REVIEW

File: packages/terra-clinical-item-collection/src/_CreateListView.jsx [hunk 1]
```diff
@@ -1,13 +1,30 @@
...
 import ItemView from 'terra-clinical-item-view';
+import styles from './ItemCollection.scss';
 
 function createListView(rows, listStyles, selectedIndex, handleSelection) {
+  const numberOfDisplays = rows[0].displays.length;
+
   const listContent = rows.map((row, rowIndex) => {
+    const { startAccessory, displays, comment, endAccessory, itemStyles } = row;
+
+    const startAccessoryContent = <div className={styles['list-accessory']}>{startAccessory}</div>;
```
Comment (jmsv6d) [inline packages/terra-clinical-item-collection/src/_CreateListView.jsx]: So this always adds a div with that 40px width. What if none of the items in the list have an accessory? Should they all still have the empty 40px in front?

Comment (emilyrohrbough) [reply packages/terra-clinical-item-collection/src/_CreateListView.jsx]: hmm good point. I made some changes in my last commit that simplified the code that took that checkout. (I knew there was a reason I had it initially)  I'll revert those changes.

**Comment by jmsv6d** [comment]
I think this looks good to fix the inconsistencies. I just think that it's kinda weird that we're just using the first row to determine if accessories should be displayed. But maybe a separate issue needs to be logged for that.

**Comment by emilyrohrbough** [comment]
@jmsv6d The use of the first row to determine the layout was something @mjhenkes and I discussed during the design of this component to determine the number of table columns dynamically. I agree its a bit weird though

**Comment by emilyrohrbough** [comment]
@PaulSolDev @tm4027 This is ready for functional verification.

**Comment by PaulSolDev** [comment]
Appears correct in IE 10, 11, and Edge.

**Comment by leemurdock** [comment]
Checks out in Safari.

**Comment by emilyrohrbough** [comment]
@PaulSolDev @tm4027 @mikertr @leemurdock  what validations are left on this?

**Comment by emilyrohrbough** [comment]
@PaulSolDev @mikertr I restarted the heroku deployment. Here is the link: https://terra-clinical-deployed-pr-190.herokuapp.com/static/#/site

**Comment by mikertr** [comment]
Looks good in macOS Chrome and Firefox.

**Pull request merged**